tests/python/rest_api/test_limits.py

```diff
@@ -138,7 +138,7 @@ def test_can_reach_tasks_limit(self):
             self._create_task()
 
         assert capture.value.status == HTTPStatus.FORBIDDEN
-        assert set(json.loads(capture.value.body)) == {self._TASK_LIMIT_MESSAGE}
+        assert -1 < json.loads(capture.value.body)["detail"].rfind(self._TASK_LIMIT_MESSAGE)
```

Comment:
Will `not  self._TASK_LIMIT_MESSAGE in capture.value.body` work? It looks simpler, isn't it?